### PR TITLE
[14.0][ADD] - New module sale_order_change_analytic_account

### PR DESCRIPTION
**Pull request by sbejaoui**
This addon allow user to update analytic account on sale orders and related journal entries.

### CODE REVIEW

**Comment by sbejaoui** [comment]
cc/ @rousseldenis

File: sale_order_change_analytic_account/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def action_open_sale_order_change_analytic_wizard(self):
```
Comment (rousseldenis) [inline sale_order_change_analytic_account/models/sale_order.py:10]: @sbejaoui This function is not called anywhere.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Could you check behaviour as no action link nor button is present on sale order ?

Could you add a readme fragment that explain a little bit more ('the analytic account is readonly on sale orders with invoices, so, this is a helper to update all already linked invoices' or something similar) ?

File: sale_order_change_analytic_account/models/sale_order.py (hunk 1 above)
Comment (sbejaoui) [reply sale_order_change_analytic_account/models/sale_order.py:10]: fixed

**Review by rousseldenis — APPROVED** [review]
Code and functional review

**Comment by rousseldenis** [comment]
The only thing is that analytic account is maybe undefined on Account Invoice level. It's only updated on Invoice lines.

Maybe something should be formalized on README. @Cedric-Pigeon What do you think ?

**Pull request merged**